### DISCUSSION TITLE
RSA导入私角出错

### DISCUSSION THREAD

**Discussion by songjinlian** [Leon406/ToolsFx#246] [Q&A] [answered]
导入RSA私钥时出错。私钥文件如下
[snakeoil-rsa.zip](https://github.com/Leon406/ToolsFx/files/14926997/snakeoil-rsa.zip)

**Comment by Leon406** [comment, marked as answer]
.key后缀目前按照二进制解析的，改下后缀名或者删除就可以了

**Comment by Leon406** [comment]
github action更新了一版
